File: qe-selenium-java/src/main/java/demos/ActivityTwo.java
package demos;

import org.openqa.selenium.firefox.*;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;

public class ActivityTwo {
    public static void main(String[] args) throws InterruptedException {
        WebDriver driver = new FirefoxDriver();

        driver.get("https://training-support.net/webelements/login-form");
        System.out.println(driver.getTitle());

        driver.findElement(By.id("username")).sendKeys("admin");
        driver.findElement(By.id("password")).sendKeys("password");
        driver.findElement(By.xpath("/html/body/div/main/div/div/div/div/div[2]/form/button")).click();
        // driver.quit();

    }

}

File: qe-selenium-python/activityTwo.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from time import sleep

with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/login-form")
    print(driver.title)

    driver.find_element(By.ID, "username").send_keys("admin")
    driver.find_element(By.ID, "password").send_keys("password")
    driver.find_element(By.XPATH, "/html/body/div/main/div/div/div/div/div[2]/form/button").click()
"""    driver.quit()
"""

